### DISCUSSION TITLE
Self Hosted Version 3.5

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#706] [Q&A]
Hello,

are there any plans to release new docker images with some new features implemented for self hosters like "dot" in the alias?

Thanks

**Comment by nguyenkims** [comment]
Hi we don't have an ETA on the new version yet. Publishing a new version requires a lot of manual testing so we only release a new version when enough people ask for it :).

**Reply by ghost** [reply]
Thanks, I do understand that this takes time.
I tried to build a docker image from several git snapshots but they all failed when initializing the database.
So please put me on the list of the ones who ask for a new version.

Thanks
